src/styles/components/_product-information.scss

```diff
@@ -0,0 +1,20 @@
+.product-information {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: rem(16px);
+  box-sizing: border-box;
```

Comment:
Same as this [comment](https://github.com/Cobda/cobda-web/pull/6/files#r571752547)